src/Stratis.Bitcoin.Features.Consensus/ProvenBlockHeaders/IRewindDataIndexRepository.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.Consensus.ProvenBlockHeaders
+{
+    /// <summary>
+    /// Interface to insert and retrieve <see cref="RewindData"/> index items from the database repository.
+    /// </summary>
+    public interface IRewindDataIndexRepository : IDisposable
+    {
+        /// <summary>
+        /// Persists key value pair for the rewind data index item.
+        /// </summary>
+        /// <param name="key">Transaction id + N (N is an index of output in a transaction).</param>
+        /// <param name="rewindDataIndex">Index of the rewind data index.</param>
+        /// <param name="cancellationToken">Cancellation token.</param>
+        Task PutAsync(string key, int rewindDataIndex, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
as pointed out in the implementation, this seems not used

Reply:
Fixed